projects/js-packages/components/components/pricing-table/styles.module.scss

```diff
@@ -0,0 +1,102 @@
+.container {
+	--padding: 32px;
+	color: var( --jp-black );
+}
+
+.table {
+	--gap: 24px;
+	position: relative;
+	z-index: 2;
+	padding: var( --padding ) 0;
+
+	.is-viewport-large & {
+		display: grid;
+		grid-template-columns: repeat( var( --columns ), 1fr );
+		grid-auto-flow: column;
+		grid-template-rows: repeat( var( --rows ), minmax( min-content, max-content ) );
+		column-gap: var( --gap );
+	}
+}
+
+.card {
```

Comment:
Looks like almost the entire logic is duplicated here and in the header, we could merge both.